docker/linksets/linksets-builder/upload_ttl.py

```diff
@@ -0,0 +1,18 @@
+import boto3
+from botocore import UNSIGNED
+from botocore.client import Config
+import os
+import utils
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+logging.info("Uploading linkset to s3")
+
+utils.fail_or_getenv('AWS_ACCESS_KEY_ID')
+utils.fail_or_getenv('AWS_SECRET_ACCESS_KEY')
+s3_bucket = utils.fail_or_getenv('S3_BUCKET')
+s3_linkset_path = utils.fail_or_getenv('S3_LINKSET_PATH')
+s3_region_name = utils.fail_or_getenv('S3_REGION')
+s3_client = boto3.client('s3', region_name=s3_region_name)
+filename = s3_linkset_path+ '/' + "ls_mb16cc.ttl"
```

Comment:
Please use the `MB2CC_LINKSET_FILE` env variable. It will be available to your container after you add the `common.env` to your dockerfile as suggested above. We need consistent naming so that cache can download the right files.

Reply:
Also, I would ideally like all output formats to be the same. At this stage I like zipped trig files. See `docker/harvest/asgs/resources/main.sh` for how I do that if you like.

Reply:
done